src/ape/api/transactions.py

```diff
@@ -183,26 +183,47 @@ def __repr__(self) -> str:
         return f"<{cls_name} {params}>"
 
     def __str__(self) -> str:
-        # NOTE: Using JSON mode for style.
-        data = self.model_dump(mode="json")
-        if len(data["data"]) > 9:
-            # only want to specify encoding if data["data"] is a string
-            if isinstance(data["data"], str):
-                data["data"] = (
+        return self.to_string()
+
+    def to_string(
+        self, calldata_repr: Optional[Union[Literal["full"], Literal["abridged"]]] = None
```

Comment:
Can this use `CalldataRepr` enum?

Reply:
It could, but I may just have to move the enum out of the core plugin then.

Reply:
I think that makes sense if you turn this into a core display configuration instead of just a config for ape-accounts

Reply:
done